### PR TITLE
Added missing C# example in tutorials -> best practices -> godot_notifications.

### PR DESCRIPTION
**Pull request by GamEsnitzhel**
There's a missing C# code example in https://docs.godotengine.org/en/latest/tutorials/best_practices/godot_notifications.html. All this does is add it in.

### CODE REVIEW

**Review by raulsntos — CHANGES_REQUESTED** [review]

File: tutorials/best_practices/godot_notifications.rst [hunk 1]
```diff
@@ -84,6 +84,26 @@ implementing a Timer-timeout loop is another option.
...
+ .. code-tab:: csharp
+
+    using Godot;
+
+    public partial class MyNode : Node
+    {
+
+        // Allows for recurring operations that don't trigger script logic
+        // every frame (or even every fixed frame).
+        public void _Process(double delta)
```
[the remark this answers is not in the document]
Comment (GamEsnitzhel) [reply tutorials/best_practices/godot_notifications.rst:96]: Yep! Thanks, I didn't catch that somehow.

[the remark this answers is not in the document]
Comment (GamEsnitzhel) [reply tutorials/best_practices/godot_notifications.rst]: Changed it to `public void _Ready()`

File: tutorials/best_practices/godot_notifications.rst [hunk 2]
```diff
@@ -84,6 +84,24 @@ implementing a Timer-timeout loop is another option.
...
 
+ .. code-tab:: csharp
+
+    using Godot;
+
+    public partial class MyNode : Node
+    {
+        // Allows for recurring operations that don't trigger script logic
+        // every frame (or even every fixed frame).
+        public void _Ready()
```
Comment (raulsntos) [inline tutorials/best_practices/godot_notifications.rst:95]: The `_Ready` method is virtual, in C# you need to use the `override` keyword when overriding methods.

```suggestion
        public override void _Ready()
```

**Review by raulsntos — APPROVED** [review]
LGTM. Thanks!

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Merged. Thanks and congrats on your first merged contribution!

**Pull request merged**